### DISCUSSION TITLE
サイドバーなどテンプレートの翻訳について

### DISCUSSION THREAD

**Discussion by potappo** [mozilla-japan/translation#620] [General]
(2022/07/23 MDN 翻訳ミートアップより)

サイドバーなど、テンプレートになっている部分の翻訳は、yari リポジトリにある KumaScript の翻訳が必要です。

https://github.com/mdn/yari/tree/main/kumascript/macros

翻訳内容を日本語コミュニティチームの人にレビューしてもらって、マージは、他の言語の PR をみると、mdn-community-engagement チームの人にお願いする形になるようです。

MDN 翻訳ミートアップでの PR：
https://github.com/mdn/yari/pull/6757

参考にした PR：
https://github.com/mdn/yari/pull/6740

**Comment by dynamis** [comment]
なるほど！ FAQ / HowTo な感じですね。取りあえず MDN, Yari ラベル付けておきます
